styles/blocks/features/features.less

```
@@ -0,0 +1,50 @@
+.features {
+    background-color: #EEEEEE;
+    display: flex;
+    justify-content: center;
+
+    @media @screen-only-s {
+        flex-direction: column;
+    }
+
+    &__list {
+        list-style: none;
+        padding: 0;
+        display: flex;
+        justify-content: space-between;
+        margin: 40px 0;
+        width: 100%;
+
+        @media @screen-only-s {
+            flex-direction: column;
+            gap: 30px;
+            margin: 30px 15px;
+        }
+    }
+
+    &__item {
+        display: flex;
+        align-items: center;
+    }
+
+    &__name {
+        max-width: 160px;
+        margin: 0 0 0 30px;
+
+        @media @screen-only-m {
+            margin: 0 0 0 10px;
+        }
+
+        @media @screen-only-s {
+            margin: 0 0 0 22px;
+        }
+    }
+
+    &__icon {
+        width: 50px;
+
+        @media @screen-only-m-s {
+            width: 40px;
+        }
+    }
+}
```

нужно добавить пустую строку в конце файла
```suggestion
}

```